### PR TITLE
All

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: 04-spatial-operations.Rmd [hunk 1]
```diff
@@ -418,19 +426,23 @@ sum(st_coordinates(cycle_hire)[, 1] %in% st_coordinates(cycle_hire_osm)[, 1])
...
-  # tm_bubbles(col = "blue", alpha = 0.5, size = 0.2) +
-  # tm_shape(cycle_hire_osm) +
-  # tm_bubbles(col = "red", alpha = 0.5, size = 0.2) +
-  # tm_scale_bar()
-library(leaflet)
-leaflet() %>%
-  addProviderTiles(providers$OpenStreetMap.BlackAndWhite) %>%
-  addCircles(data = cycle_hire) %>%
-  addCircles(data = cycle_hire_osm, col = "red")
+if (knitr::is_latex_output()){
```
Comment (Robinlovelace) [inline 04-spatial-operations.Rmd]: The magic command!

File: 08-mapping.Rmd [hunk 2]
```diff
@@ -381,7 +389,7 @@ map_nza + tm_style("cobalt")
 map_nza + tm_style("col_blind")
 ```
 
-```{r tmstyles, message=FALSE, fig.cap="Selected tmap styles: bw, classic, cobalt and col_blind (from left to right).", fig.scap="Selected tmap styles: bw, classic, cobalt and col_blind.", echo=FALSE, fig.asp=0.56}
+```{r tmstyles, message=FALSE, fig.cap="Selected tmap styles.", fig.scap="Selected tmap styles.", echo=FALSE, fig.asp=0.56}
```
Comment (Robinlovelace) [inline 08-mapping.Rmd]: Looks good. Simpler.

File: 08-mapping.Rmd [hunk 3]
```diff
@@ -559,12 +577,19 @@ source("https://github.com/Robinlovelace/geocompr/raw/master/code/08-uscolonize.
 ```
 
 ```{r animus, echo=FALSE, message=FALSE, fig.cap="Animated map showing population growth, state formation and boundary changes in the United States, 1790-2010. Animated version available online at geocompr.robinlovelace.net.", fig.scap="Animated map showing boundary changes in the United States."}
-u_animus = "https://user-images.githubusercontent.com/1825120/38543030-5794b6f0-3c9b-11e8-9da9-10ec1f3ea726.gif"
-knitr::include_graphics(u_animus)
+u_animus_html = "https://user-images.githubusercontent.com/1825120/38543030-5794b6f0-3c9b-11e8-9da9-10ec1f3ea726.gif"
```
Comment (Robinlovelace) [inline 08-mapping.Rmd]: Nice!

File: 09-gis.Rmd [hunk 4]
```diff
@@ -48,30 +48,30 @@ The 'CLI vs GUI' debate can be adversial but it does not have to be; both option
...
-The advantages of a good CLI such as that provided by R (and enhanced by IDEs such as RStudio) are numerous.
+The advantages of a good CLI\index{command-line interface} such as that provided by R (and enhanced by IDEs\index{IDE} such as RStudio\index{RStudio}) are numerous.
 A good CLI
 
-- facilitates the automation of repetitive tasks. 
-- enables transparency and reproducibility, the backbone of good scientific practice and data science.
-- encourages software development by providing tools to modify existing functions and implement new ones.
-- helps develop future-proof programming skills which are in high demand in many disciplines and industries.
-- is user-friendly and fast, allowing an efficient workflow.
+- Facilitates the automation of repetitive tasks.
```
Comment (Robinlovelace) [inline 09-gis.Rmd]: This is the old style. But we can tidy up later 🚿

File: 12-transport.Rmd [hunk 5]
```diff
@@ -261,9 +264,9 @@ od_top5 = bristol_od %>%
 od_top5 %>% 
   knitr::kable(
     caption = paste("Sample of the origin-destination data stored in the data",
-                    "frame object `bristol_od`. These represent the top 5 most", 
+                    "frame object `bristol\\_od`. These represent the top 5 most",
```
Comment (Robinlovelace) [inline 12-transport.Rmd]: @Nowosad how does that look in the html?

**Comment by Robinlovelace** [comment]
This looks good to me. It's passing. So please merge as you see fit @Nowosad + @jannes-m.

**Comment by Robinlovelace** [comment]
OK I'll take a look:

image

File: 12-transport.Rmd (hunk 5 above)
Comment (Nowosad) [reply 12-transport.Rmd]: Good catch @Robinlovelace - it does not look perfectly... Have you got any idea how to improve it?

Comment (Robinlovelace) [reply 12-transport.Rmd]: Hey @Nowosad I've seen this before and there are other captions that had `_`. Would just change it to


https://github.com/Robinlovelace/geocompr/blob/b90787e016e9edda841bee159c63659c12d2d078/12-transport.Rmd#L266-L268

Comment (Robinlovelace) [reply 12-transport.Rmd]: Proposed fix: https://github.com/Robinlovelace/geocompr/pull/433/files

**Pull request merged**